integration-test/background/click-to-load-facebook.js

```diff
@@ -1,23 +1,67 @@
 const harness = require('../helpers/harness')
+const { logPageRequests } = require('../helpers/requests')
 
 const testSite = 'https://privacy-test-pages.glitch.me/privacy-protections/click-to-load/'
+const facebookDomains = new Set(['facebook.com', 'facebook.net'])
+
 let browser
 let bgPage
 let teardown
 
+function summariseFacebookRequests (requests) {
+    const facebookSDKRedirect = { checked: false, alwaysRedirected: true }
+    // Note: Requests to the SDK are not included in counts.
+    let requestCount = 0
+    let allowCount = 0
+    let blockCount = 0
+
+    for (const request of requests) {
+        if (facebookDomains.has(request.url.hostname.split('.').slice(-2).join('.'))) {
+            if (request.url.pathname === '/en_US/sdk.js') {
+                facebookSDKRedirect.alwaysRedirected = (
+                    facebookSDKRedirect.alwaysRedirected &&
```

Comment:
smart, probably too smart, drop a comment that this sdk is loading multiple times on the page so that's why we need this check

Reply:
I think that this is made pretty clear by the variable names `facebookSDKRedirect.alwaysRedirected` and `facebookSDKRedirect.checked`. Our tests need to know that the SDK request was made at least one and if requests to the SDK were redirected every time it was made. I figure that knowing just if the last request to the SDK was redirected doesn't tell us much from a privacy respect.

Reply:
fair enough 👍